File: kids greatest candies/O(n).cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
        vector<bool> booleans;
        int max = candies.at(0);
        for (int i = 0; i < candies.size(); i++) {
            if (candies.at(i) > max) {
                max = candies.at(i);
            }
        }

        for (int i = 0; i < candies.size(); i++) {
            booleans.push_back(max <= candies.at(i) + extraCandies);
        }

        return booleans;
    }
};

File: Nth Finbonacci/O(n).py
class Solution:
    def tribonacci(self, n: int) -> int:
        F = []
        F.append(0)
        F.append(1)
        F.append(1)
        if n == 0:
            return F[0]
        while len(F) <= n:
            F.append(F[len(F) - 3] + F[len(F) - 2] + F[len(F) - 1])
        return F[len(F) - 1]